models/DataObject/ClassDefinition/Data/Classificationstore.php

```diff
@@ -332,7 +332,7 @@ public function getDataFromEditmode(
                             $keyConfig
                         );
 
-                        $dataFromEditMode = $dataDefinition->getDataFromEditmode($value);
+                        $dataFromEditMode = $dataDefinition->getDataFromEditmode($value, $object);
```

Comment:
Additionally, the object can be useful for operations in the getDataFromEditmode, as it is passed in normal data-objects fields